docs/docs/adapters/kysely.md

```diff
@@ -0,0 +1,275 @@
+  dialect: new PostgresDialect({
+    pool: new Pool({
+      host: process.env.DATABASE_HOST,
+      database: process.env.DATABASE_NAME,
+      user: process.env.DATABASE_USER,
+      password: process.env.DATABASE_PASSWORD,
+    }),
+  }),
+})
+```
+
+### Create migrations
+
+This schema is based on the NextAuth.js main [schema](/adapters/models).
+
+```ts title="db/migrations/001_create_db.ts"
+import { Kysely, sql } from "kysely"
+
+export async function up(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .createTable("User")
+    .addColumn("id", "uuid", (col) =>
+      col.primaryKey().defaultTo(sql`gen_random_uuid()`)
+    )
+    .addColumn("name", "text")
+    .addColumn("email", "text", (col) => col.unique())
+    .addColumn("emailVerified", "timestamptz", (col) =>
+      col.defaultTo(sql`NOW()`)
+    )
+    .addColumn("image", "text")
+    .execute()
+
+  await db.schema
+    .createTable("Account")
+    .addColumn("id", "uuid", (col) =>
+      col.primaryKey().defaultTo(sql`gen_random_uuid()`)
+    )
+    .addColumn("userId", "uuid", (col) =>
+      col.references("User.id").onDelete("cascade").notNull()
+    )
+    .addColumn("type", "text", (col) => col.notNull())
+    .addColumn("provider", "text", (col) => col.notNull())
+    .addColumn("providerAccountId", "text", (col) => col.notNull())
+    .addColumn("refresh_token", "text")
+    .addColumn("access_token", "text")
+    .addColumn("expires_at", "bigint")
+    .addColumn("token_type", "text")
+    .addColumn("scope", "text")
+    .addColumn("id_token", "text")
+    .addColumn("session_state", "text")
+    .addColumn("oauth_token_secret", "text")
+    .addColumn("oauth_token", "text")
+    .execute()
+
+  await db.schema
+    .createTable("Session")
+    .addColumn("id", "uuid", (col) =>
+      col.primaryKey().defaultTo(sql`gen_random_uuid()`)
+    )
+    .addColumn("userId", "uuid", (col) =>
+      col.references("User.id").onDelete("cascade").notNull()
+    )
+    .addColumn("sessionToken", "text", (col) => col.notNull().unique())
+    .addColumn("expires", "timestamptz", (col) => col.notNull())
+    .execute()
+
+  await db.schema
+    .createTable("VerificationToken")
+    .addColumn("identifier", "text", (col) => col.notNull())
+    .addColumn("token", "text", (col) => col.notNull().unique())
+    .addColumn("expires", "timestamptz", (col) => col.notNull())
+    .execute()
+
+  await db.schema
+    .createIndex("Account_userId_index")
+    .on("Account")
+    .column("userId")
+    .execute()
+
+  await db.schema
+    .createIndex("Session_userId_index")
+    .on("Session")
+    .column("userId")
+    .execute()
+}
+
+export async function down(db: Kysely<any>): Promise<void> {
+  await db.schema.dropTable("Account").ifExists().execute()
+  await db.schema.dropTable("Session").ifExists().execute()
+  await db.schema.dropTable("User").ifExists().execute()
+  await db.schema.dropTable("VerificationToken").ifExists().execute()
+}
+```
+
+### Create migration runner
```

Comment:
same as above

Reply:
Agreed, removed in favor of pointing to the Kysely docs. It was next.js specific anyways and with the Auth.js rebrand, doesn't make sense anymore.